src/orgwarden/__main__.py

```diff
@@ -23,6 +23,15 @@ def list_repos(
             show_default=False,
         ),
     ],
+    gh_pat: Annotated[
```

Comment:
Would we ever want to run `OrgWarden` without a PAT? (I can't think of a reason why we would).